recipes/farm-haystack/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "farm-haystack" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3ea141aea5e3c97b2eebd66cec3e7ba0b3527b055f09f3a3c74be60af400ceb2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```